src/backend/src/controllers/work-packages.controllers.ts

```diff
@@ -43,15 +44,16 @@ export default class WorkPackagesController {
 
       const wbsString: string = await WorkPackagesService.createWorkPackage(
         user,
-        projectWbsNum,
         name,
         crId,
         stage,
         startDate,
         duration,
         blockedBy,
         expectedActivities,
-        deliverables
+        deliverables,
+        projectLead,
```

Comment:
Oh fuck, miscommunication, we aren't adding projectLead/Manager to this endpoint after a conversation with walter/sofia, they said it makes more sense to wait for activation like we currently do